webapp/channels/src/components/dot_menu/post_reminder_submenu.tsx

```diff
@@ -136,14 +143,18 @@ function PostReminderSubmenu(props: Props) {
                 key={`remind_post_options_${postReminder}`}
                 labels={labels}
                 trailingElements={trailingElements}
-                onClick={() => handlePostReminderMenuClick(postReminder)}
+                onClick={() => handlePostReminder(postReminder)}
+                onKeyDown={(event) => handleKeydown(event, postReminder)}
             />
         );
     });
 
+    const header = formatMessage({id: 'post_info.post_reminder.sub_menu.header', defaultMessage: 'Set a reminder for:'});
```

Comment:
Minor nit, but could you move the `.replace` call up here? I think it helps to keep the code clear by having those both in the same place